src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -363,9 +363,9 @@ const Component = compose(
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
                     days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
+                      '--')
```

Comment:
ok 155 at 0, had to edit the url

https://portal.gdc.cancer.gov/search/c?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22%3E%3D%22,%22content%22:%7B%22field%22:%22cases.diagnoses.days_to_last_follow_up%22,%22value%22:%5B0%5D%7D%7D,%7B%22op%22:%22%3C%3D%22,%22content%22:%7B%22field%22:%22cases.diagnoses.days_to_last_follow_up%22,%22value%22:%5B0%5D%7D%7D%5D%7D&facetTab=cases